gdal/swig/python/osgeo/utils/gdal_calc.py

```diff
@@ -457,23 +665,32 @@ def main(argv):
 
     parser.add_option("--outfile", dest="outF", help="output file to generate or fill", metavar="filename")
     parser.add_option("--NoDataValue", dest="NoDataValue", type=float, help="output nodata value (default datatype specific value)", metavar="value")
-    parser.add_option("--type", dest="type", help="output datatype, must be one of %s" % list(DefaultNDVLookup.keys()), metavar="datatype")
+    parser.add_option("--hideNoData", dest="hideNoData", action="store_true", help="ignores the NoDataValues of the input rasters", metavar="value")
+    parser.add_option("--type", dest="type", help="output datatype, must be one of %s" % GDALDataTypeNames, metavar="datatype")
```

Comment:
the choice could probably be implemented with a ``choices`` argument

Reply:
done